ghost/member-attribution/lib/service.js

```diff
@@ -63,7 +111,15 @@ class MemberAttributionService {
      */
     getEventAttribution(eventModel) {
         if (eventModel.get('attribution_type') === null) {
-            return null;
+            return {
+                id: null,
+                url: null,
+                title: null,
+                type: null,
+                referrerSource: null,
+                referrerMedium: null,
+                referrerUrl: null
+            };
         }
```

Comment:
In the activity feed we return this, while not in the activity feed we return `getSubscriptionCreatedAttribution`, which is 
```js
const attribution = this.attributionBuilder.build({
            id: subscriptionCreatedEvent.get('attribution_id'),
            url: subscriptionCreatedEvent.get('attribution_url'),
            type: subscriptionCreatedEvent.get('attribution_type'),
            referrerSource: subscriptionCreatedEvent.get('referrer_source'),
            referrerMedium: subscriptionCreatedEvent.get('referrer_medium'),
            referrerUrl: subscriptionCreatedEvent.get('referrer_url')
        });
        return await attribution.fetchResource();
```

Reply:
Ah nice catch! I was not sure if it was a good idea to let the builder run here if type was `null`, but think the check for type in line below should cover it, pushing this, thanks.